gcp_variant_transforms/beam_io/vcf_header_io_test.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2017 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for vcfio_header_io module."""
+
+import os
+import unittest
+
+import vcf
+
+from apache_beam.testing.util import assert_that
+from apache_beam.testing.util import equal_to
+import apache_beam.io.source_test_utils as source_test_utils
+from apache_beam.testing.test_pipeline import TestPipeline
+from gcp_variant_transforms.beam_io.vcf_header_io import _VcfHeaderSource
+from gcp_variant_transforms.beam_io.vcf_header_io import ReadVcfHeaders
+from gcp_variant_transforms.beam_io.vcf_header_io import VcfHeader
+from gcp_variant_transforms.testing import temp_dir
+from gcp_variant_transforms.testing import testdata_util
+
+_SAMPLE_HEADER_LINES = [
+    '##fileformat=VCFv4.2\n',
+    '##INFO=<ID=NS,Number=1,Type=Integer,Description="Number samples">\n',
+    '##INFO=<ID=AF,Number=A,Type=Float,Description="Allele Frequency">\n',
+    '##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">\r\n',
+    '##FORMAT=<ID=GQ,Number=1,Type=Integer,Description="Genotype Quality">\n',
+    '#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	Sample1	Sample2\r\n',
+]
+
+_SAMPLE_TEXT_LINES = [
```

Comment:
nit: I think this can just be two lines since we're not actually reading/verifying it.

Reply:
Done.